digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectValidator.scala

```diff
@@ -502,14 +538,19 @@ object ProjectValidator {
       }
     }
 
-    def getLastValidLinkForParts = {
-      seq.filter(r => r.status != LinkStatus.Terminated && r.endAddrMValue == seq.maxBy(_.endAddrMValue).endAddrMValue)
+    val continuityValidations: Seq[Seq[ValidationErrorDetails] =>  Seq[ValidationErrorDetails]] = Seq(
+      checkContinuityBetweenParts,
```

Comment:
Instead of all the methods has as parameters errors they should only have ` (project: RoadAddressProject, projectLinks: Seq[ProjectLink])`. Then those methods don't need to be nested of the `validateProject` method.